packages/athena/sodasql/dialects/athena_dialect.py

```diff
@@ -23,14 +23,13 @@ def get_aws_credentials_optional(parser: Parser):
         access_key_id = parser.get_str_optional_env('access_key_id')
         role_arn = parser.get_str_optional_env('role_arn')
         profile_name = parser.get_str_optional_env('profile_name')
-        if access_key_id or role_arn or profile_name:
```

Comment:
@Tonkonozhenko  Since we need at least some of these, isn't it better to leave the if condition, and log error on else?

Reply:
We don't need any of these if we use IRSA, instance profile or default profile, that's why I removed it